Gemfile.lock

```diff
@@ -828,4 +828,4 @@ RUBY VERSION
    ruby 2.3.7p456
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```

Comment:
you have to uninstall bundle 2.1.4 and install bundler 1.17.3
From the top of my head, I Think:
gem uninstall bundler
gem install bundler -v 1.17.3

if you then do bundle install you will see this change disappear :+1:

Reply:
Turns out I had both versions installed. Removed 2.1.4 :+1: